src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -246,9 +246,9 @@
                 "DIV_ASSIGN", "BOR_ASSIGN", "SL_ASSIGN", "ASSIGN", "BSR_ASSIGN", "BAND_ASSIGN",
                 "PLUS_ASSIGN", "MINUS_ASSIGN", "SR_ASSIGN", "STAR_ASSIGN", "BXOR_ASSIGN",
                 "MOD_ASSIGN",
-                // state of the configuration when test was made until
+                // COLON token ignored in check config, explained in
                 // https://github.com/checkstyle/checkstyle/issues/4122
```

Comment:
other ignores have text in file and not in issue.
@romani What is your opinion here?

Reply:
If simple to explain text is ok, it reason is long link to ticket or comment is better